admin_manual/configuration_server/language_configuration.rst

```diff
@@ -36,3 +36,34 @@ this value can be set to ``true`` instead of a language code.
 .. note:: Please check `Transifex language codes
    <https://www.transifex.com/explore/languages/>`_ for the list of valid language
    codes.
+
+Default locale
+--------------
+The locale is used to define how dates and other formats are displayed. Nextcloud
+should automatically pick an appropriated locale based on your current language.
+Users can modify their locale inside their settings panel.
+If that does not work properly or if you want to make sure that Nextcloud always
+starts with a given locale, you can set a **default_locale** parameter in the 
+:file:`config/config.php`.
+
+.. note:: The default_locale paramenter is only used when the user hasn't configured
+   own locale preferences.
+
+::
+
+    "default_locale" => "en_US",
+
+Force locale
+--------------
+
+If you want to force a specific locale, users will no longer be able to change
+their locale in the personal settings. You can set a **force_locale** parameter
+in the :file:`config/config.php`.
+
+'::
+
+    "force_locale" => "en_US",
+
+.. note:: Please check `Transifex language codes
```

Comment:
Copy and Paste leftover? ;) 

What locale should one check? The moment js locales?

Reply:
Well, the data are provided by moment indeed, so we should probably check moment instead, you're right!

Reply:
Anything better than https://github.com/moment/moment/tree/develop/locale ?

Reply:
I think i would prefer to link to the stable, used version: https://github.com/moment/moment/tree/2.18.1/locale

But then we would have to update the link every time we update moment.js

Reply:
Does it matter much in the end ? Maybe remove the whole note ?